scripts/in_container/configure_environment.sh

```diff
@@ -48,3 +49,20 @@ else
     echo "In it to make breeze source the variables automatically for you"
     echo
 fi
+
+
+if [[ -d "${AIRFLOW_BREEZE_CONFIG_DIR}" && \
+    -f "${AIRFLOW_BREEZE_CONFIG_DIR}/${TMUX_CONF_FILE}" ]]; then
+    pushd "${AIRFLOW_BREEZE_CONFIG_DIR}" >/dev/null 2>&1 || exit 1
+    echo
+    echo "Using ${TMUX_CONF_FILE} from ${AIRFLOW_BREEZE_CONFIG_DIR}"
+    echo
+     # shellcheck disable=1090
+    cp "${AIRFLOW_BREEZE_CONFIG_DIR}/${TMUX_CONF_FILE}" ~
```

Comment:
Would it be possible to `ln -s` instead so changes in the container can be kept?

Reply:
> Would it be possible to `ln -s` instead so changes in the container can be kept?

Good point!

Reply:
Thanks for the suggestion @uranusjr.
changes has been done